shared/src/msi-installer/shared/EnvironmentVariables.wxs

```diff
@@ -12,8 +12,8 @@
   
         <Environment Id="COR_PROFILER_PATH_32" Name="COR_PROFILER_PATH_32" Action="set" Permanent="no" System="yes" Value="[INSTALLFOLDER]win-x86\Datadog.Trace.ClrProfiler.Native.dll" Part="all" />
         <Environment Id="CORECLR_PROFILER_PATH_32" Name="CORECLR_PROFILER_PATH_32" Action="set" Permanent="no" System="yes" Value="[INSTALLFOLDER]win-x86\Datadog.Trace.ClrProfiler.Native.dll" Part="all" />
-  
         <?if $(var.Win64) = yes ?>
+          <Environment Id="PATH" Name="PATH" Action="set" Permanent="no" System="yes" Value="[INSTALLFOLDER]win-x64" Part="last" />
```

Comment:
For others wondering `Part="last"` is the magic that makes this a suffix instead of completely replacing the value

Reply:
Is it something we should let the customer decide whether to do it or not? 
Also, I assume WIX is clever enough not to add the path multiple times, right?

Reply:
> Is it something we should let the customer decide whether to do it or not?

Yeah, it might be nice/a good idea to give them the option to disable that in the installer 🤔 

> Also, I assume WIX is clever enough not to add the path multiple times, right?

Good question. Related, does it _remove_ it when you uninstall it?

Reply:
> Is it something we should let the customer decide whether to do it or not?

I don't think it matters. We only have dd-dotnet.exe in there, the risk of name collision is close to zero.

> Related, does it remove it when you uninstall it?

Good question, I'll test it.

Reply:
I confirm that uninstalling removes it from path.